# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../djv/djv-0.8.2-disable-third-party.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Disable to build the included third-party applications
Origin: OpenSDE (http://opensde.org)

djv ships with all needed third-party libraries and applications but we                                                       # disabled to build/use them because we are using those provided by the
system.

diff -ruN djv-0.8.2-orig/CMakeLists.txt djv-0.8.2/CMakeLists.txt
--- djv-0.8.2-orig/CMakeLists.txt	2009-08-01 00:32:48.000000000 +0200
+++ djv-0.8.2/CMakeLists.txt	2011-04-15 14:38:19.176077819 +0200
@@ -207,7 +207,7 @@
 
 # Uncomment the following line to use system versions of third-party software:
 
-#set(djv_third_party_use_system true)
+set(djv_third_party_use_system true)
 
 if (WIN32)
 
@@ -262,20 +262,20 @@
     set(
       djv_third_party
 #      mesa
-      glew
-      zlib
-      libjpeg
-      libpng
-      libtiff
-      ffmpeg
-      libquicktime
-      fltk
-      ilmbase
+#      glew
+#      zlib
+#      libjpeg
+#      libpng
+#      libtiff
+#      ffmpeg
+#      libquicktime
+#      fltk
+#      ilmbase
 #      ctl
-      openexr
+#      openexr
 #      openexr_ctl
 #      openexr_viewers
-      portaudio
+#      portaudio
     )
 
   else (NOT djv_build_minimal)
